packages/components/icons/src/Gift.tsx

```diff
@@ -3,6 +3,7 @@ import { generateIcon } from '@contentful/f36-icon';
 
 export const Gift = generateIcon({
   name: 'Gift',
+  viewBox: '0 0 18 18',
```

Comment:
🤔 why do we need this viewBox?

Reply:
because it was like that in v3 and it wasn't converted properly